uo_sys_framework/Core/cfgFunctions/JiP/fn_Teleport.sqf

```diff
@@ -44,16 +45,19 @@ if (!((_target call EFUNC(Core,Alive)) && {((_target isEqualto vehicle _target)
         };
     };
 };
+LOG_1("Jip Teleport to %1",_target);
 if (!isNull(_target)) then {
-    if (_target isEqualto vehicle _target) then {
+    if !(_target isEqualto vehicle _target) then {
         private _moveInSuccess = player moveInAny (vehicle _target);
-        [{!(isNil (this select 0))}, {
+        [{!(isNil (_moveInSuccess))}, {
```

Comment:
is `_moveInSuccess` defined? Condition checks are in a different scope/thread than the passed parameters. This might need to be _this select 3 select 0

Reply:
The condition of a waitUntil should still be on the same thread.

Reply:
_this select 0 caused undefined error

Reply:
See slack for details on unintended behavior, but teleporting to group leader in a vehicle is working.